docs/core/install/linux-fedora.md

```diff
@@ -12,32 +12,20 @@ ms.date: 01/06/2021
 
 [!INCLUDE [linux-intro-sdk-vs-runtime](includes/linux-intro-sdk-vs-runtime.md)]
 
-[!INCLUDE [linux-install-package-manager-x64-vs-arm](includes/linux-install-package-manager-x64-vs-arm.md)]
```

Comment:
This doc note includes 2 pieces: a note that only x64 is supported (not arm64) and alternate ways to install .NET. The first part of the note is not completely valid: both arm64 and x64 are available via the Fedora package manager repository. I decided to inline the relevant part, but please let me know if I should do something else instead.